charts/ai-software-templates/chatbot/0.1.0/templates/tests/test-connection-app.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Pod
```

Comment:
Should this be a one off Job rather than Pod?

Reply:
TBH I kind of took that as example from other charts already in the openshift-charts. I've tried to make it a job but it failes as I think it needs to run on a specific SA (has permission error).

Reply:
The SA shouldn't matter too much if it's just running `wget`, but I'll be honest and say I'm not sure how `helm test` works